app/src/main/java/com/github/onlynotesswent/ui/user/EditProfile.kt

```diff
@@ -130,109 +131,111 @@ fun EditProfileScreen(
-                                      FileType.PROFILE_PIC_JPEG,
-                                  )
                                 }
-                              }
-                            },
-                            onFailure = { exception ->
-                              val errorMessage =
-                                  when (exception) {
-                                    is UserRepositoryFirestore.UsernameTakenException ->
-                                        "Username is already taken. Please choose a different one."
-                                    else -> "Oops! Something went wrong. Please try again later."
-                                  }
-                              Toast.makeText(localContext, errorMessage, Toast.LENGTH_SHORT).show()
-                              userNameError.value =
-                                  exception is UserRepositoryFirestore.UsernameTakenException
-                            })
-                      },
-                      modifier = Modifier.testTag("saveButton"),
-                      enabled = saveEnabled.value) {
-                        Icon(
-                            imageVector = Icons.Default.Check,
-                            contentDescription = "Save Note",
-                            tint = MaterialTheme.colorScheme.onSurface)
-                      }
-                })
+                              },
+                              onFailure = { exception ->
+                                val errorMessage =
+                                    when (exception) {
+                                      is UserRepositoryFirestore.UsernameTakenException ->
+                                          "Username is already taken. Please choose a different one."
+                                      else -> "Oops! Something went wrong. Please try again later."
+                                    }
+                                Toast.makeText(localContext, errorMessage, Toast.LENGTH_SHORT)
+                                    .show()
+                                userNameError.value =
+                                    exception is UserRepositoryFirestore.UsernameTakenException
+                              })
+                        },
+                        modifier = Modifier.testTag("saveButton"),
+                        enabled = saveEnabled.value) {
+                          Icon(
+                              imageVector = Icons.Default.Check,
+                              contentDescription = "Save Note",
+                              tint = MaterialTheme.colorScheme.onSurface)
+                        }
+                  })
+
+              HorizontalDivider(
+                  color = MaterialTheme.colorScheme.onSurface.copy(alpha = 0.12f),
+                  thickness = 0.5.dp)
+            }
           },
           content = { paddingValues ->
             LazyColumn(
```

Comment:
Not really in the scope of this PR, but since you are modifying UI elements:
Could you add some vertical padding to the lazy column? It would make the scroll much nicer for thinner screens.